src/features/Player/PlayerCard/PlayerCard.tsx

```diff
@@ -0,0 +1,131 @@
+import { useEffect, useMemo, useRef, useState } from 'react';
```

Comment:
해당 페이지에 ESLint의 exhaustive-deps 규칙 경고가 있는데, 이렇게 수정하는건 어떠실까요..?
```suggestion
import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
```